Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1724,6 +1719,12 @@ UpdateFspConfig (
     Fspscfg->GraphicFusaConfigEnable       = 1;            // Fusa Graphics Configuration
     Fspscfg->OpioFusaConfigEnable          = 1;            // Fusa Opio Configuration
     Fspscfg->PsfFusaConfigEnable           = 1;            // Fusa Psf Configuration
+    if (!DebugCodeEnabled()) {
+      Fspscfg->SciPinMuxEnable                                  = 0x0;
+    }
+    else{
+      Fspscfg->SciPinMuxEnable                                  = 0x1;
+    }

```

Comment:
The SciPinMux can be configured by the value of PlatformFeatures->DebugConsent.
```
if (PlatformFeatures->DebugConsent != 0) {
  Fspscfg->SciPinMuxEnable                                  = 0x1;
} else {
  Fspscfg->SciPinMuxEnable                                  = 0x0;
}
```
If FUSA SKU only, we need to check `Fspscfg->IsFusaSupported` condition as well.